src/options/utils/render-throttled.js

```diff
@@ -0,0 +1,31 @@
+const MAX_BATCH_DURATION = 50;
+const MAX_BATCH_COUNT = 50;
+const queue = [];
+let startTime = performance.now();
+let rAF = 0;
+
+export default function throttle(component) {
```

Comment:
I think lazy loading is much simpler to achieve this, we don't have to render the invisible scripts in such a complex way.
Also we may have to find a new way to set execution orders, drag'n'drop is not easy when there are so much scripts.

Reply:
Are you sure there's something much simpler than this already very simple code? Until this moment I thought there couldn't be anything simpler. Compared to all the rest of the extension, especially. I don't know Vue so it sounds to me like you're exaggerating. Also, I wouldn't say 100 scripts is "so much", half of them are disabled and I enable them occasionally, in order not to crowd the popup menu where I hide the disabled script as I always did in Tampermonkey.

Reply:
As for drag'n'drop it's quite easy on a big screen and a custom theme I have that displays the scripts in up to 3 columns depending on the display width.

Reply:
FWIW I never needed to adjust execution order in 10+ years of using and 5 years writing userscripts.

Reply:
You probably just dislike any code that is too "close-to-metal" judging by the other "hackish" PR. While I can sympathize with the sentiment, unfortunately such things are still occasionally needed for efficient rendering when there's a bit more data than in a demo app.